decode_symphonia/src/lib.rs

```diff
@@ -207,7 +209,9 @@ impl Decoder for SymphoniaDecoder {
                 track_id: None,
             },
         ) {
-            Ok(_res) => {}
+            Ok(res) => {
+                self.seek_diff = (res.required_ts - res.actual_ts) as usize;
```

Comment:
At least a debug assertions should verify that the assumption `actual <= required` is valid. It only works for `SeekMode::Accurate` which should probably be mentioned here.